yt_dlp/extractor/ntvde.py

```diff
@@ -1,29 +1,30 @@
 import re
 
 from .common import InfoExtractor
-from ..compat import compat_urlparse
 from ..utils import (
     int_or_none,
     js_to_json,
-    parse_duration,
+    url_or_none,
 )
 
+from ..utils.traversal import traverse_obj
+
 
 class NTVDeIE(InfoExtractor):
     IE_NAME = 'n-tv.de'
     _VALID_URL = r'https?://(?:www\.)?n-tv\.de/mediathek/videos/[^/?#]+/[^/?#]+-article(?P<id>.+)\.html'
```

Comment:
I know the PR didn't touch this code, but I need to fix
```suggestion
    _VALID_URL = r'https?://(?:www\.)?n-tv\.de/mediathek/videos/[^/?#]+/[^/?#]+-article(?P<id>[^/?#]+)\.html'
```